BitwardenShared/UI/Auth/StartRegistration/CheckEmail/CheckEmailView.swift

```diff
@@ -0,0 +1,83 @@
+import SwiftUI
+
+// MARK: - CheckEmailView
+
+/// A view that allows the user to create an account.
+///
+struct CheckEmailView: View {
+    // MARK: Properties
+
+    /// An object used to open urls from this view.
+    @Environment(\.openURL) private var openURL
+
+    /// The store used to render the view.
+    @ObservedObject var store: Store<CheckEmailState, CheckEmailAction, Void>
+
+    // MARK: View
+
+    var body: some View {
+        VStack(spacing: 16) {
+            VStack(alignment: .center, spacing: 0) {
+                Image(decorative: Asset.Images.checkEmail)
+                    .foregroundColor(Asset.Colors.primaryBitwarden.swiftUIColor)
+                    .frame(maxWidth: .infinity)
+                    .padding(.vertical, 32)
+
+                Text(Localizations.checkYourEmail)
+                    .styleGuide(.title2)
+                    .foregroundColor(Asset.Colors.textPrimary.swiftUIColor)
+                    .frame(maxWidth: .infinity)
+                    .padding(.bottom, 8)
+
+                Text(.init(store.state.headelineTextBoldEmail))
+                    .styleGuide(.headline)
+                    .multilineTextAlignment(.center)
+                    .padding(.bottom, 20)
+                    .padding(.horizontal, 34)
+                    .tint(Asset.Colors.textPrimary.swiftUIColor)
+                    .foregroundColor(Asset.Colors.textPrimary.swiftUIColor)
```

Comment:
I used tint here to apply the color to the MarkUp Link. Without the tint it will stay blue as highlight.
<img width="417" alt="image" src="https://github.com/user-attachments/assets/2eeab013-5836-44f5-a49b-af882c3a89c7">

Reply:
Ah, that makes more sense. In the places where I commented, could you specify both `tint` and `foregroundStyle` so that we have a color set for the non-link text as well (it looks like it is here but isn't in some of the other places I commented)?

Reply:
Yes, I'll add in the other places 👍